### PR TITLE
feat: add variable for aliyun theme of table

### PR DESCRIPTION
**Pull request by hi-zhaoyeqing**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (add variable for aliyun theme of table)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |feat: add variable for aliyun theme of table|
| 🇨🇳 Chinese |新功能：为table组件添加阿里云主题变量|

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by vthinkxie — APPROVED** [review]
lgtm

File: components/style/themes/default.less [hunk 1]
```diff
@@ -627,6 +627,8 @@
 @table-header-sort-bg: @background-color-base;
 @table-body-sort-bg: #fafafa;
 @table-row-hover-bg: @background-color-light;
+@table-selection-column-padding-left: @padding-xs;
+@table-selection-column-padding-right: @padding-xs;
```
Comment (afc163) [inline components/style/themes/default.less:631]: @table-selection-column-width 应该能满足需求了吧？

File: components/table/style/index.less [hunk 2]
```diff
@@ -102,7 +102,7 @@
           position: absolute;
           top: 50%;
           right: 0;
-          width: 1px;
+          width: @table-header-cell-split-width;
```
[the remark this answers is not in the document]
Comment (qdzhaoxiaodao) [reply components/table/style/index.less:105]: 改成0px

[the remark this answers is not in the document]
Comment (qdzhaoxiaodao) [reply components/table/style/index.less:105]: image

File: components/style/themes/default.less (hunk 1 above)
Comment (qdzhaoxiaodao) [reply components/style/themes/default.less]: 用@table-selection-column-width试了一下，可以的，谢谢

File: components/table/style/index.less (hunk 2 above)
[the remark this answers is not in the document]
Comment (afc163) [reply components/table/style/index.less:105]: 是不是 `@table-header-cell-split-color: transparent` 就够了？

[the remark this answers is not in the document]
Comment (qdzhaoxiaodao) [reply components/table/style/index.less:105]: 可以满足

**Pull request closed without merging**